.github/workflows/ci-elasticsearch.yml

```diff
@@ -9,6 +9,9 @@ on:
 jobs:
   elasticsearch:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        version: [5.6.16, 6.8.2, 7.3.0]
```

Comment:
a bit of future proofing - let's define matrix as two values per iteration, a full version (5.6.16) and major-only version (e.g. 5.x), and then define `name` for the job using only the major version (example: https://futurestud.io/tutorials/github-actions-customize-the-job-name). Otherwise the jobs get names like `elasticsearch (5.6.16)`, which must match exactly when configuring the required checks (if we bump the version we'd need to re-configure the repo).

Reply:
That's a great idea. Is to make sure that we always test against the latest stable version of ES for each major version ?

Reply:
Well, I assume at some point we may want to change the exact version being tested, like go from 5.6.16 to 5.7 or so. Right now, if we did that, that would change the job name and leave a required check that could never be satisfied. So it's better to have stable job names.